kolibri_explore_plugin/assets/src/components/TagRow.vue

```diff
@@ -0,0 +1,203 @@
+<template>
+
+  <b-container fluid>
+    <h4 class="mx-1 mx-lg-3 mx-md-2">
+      {{ label }}:
+    </h4>
+    <b-button-group class="mx-1 mx-lg-3 mx-md-2">
+      <b-button
+        :disabled="!hasPrevious"
+        variant="outline-light"
+        @click="goPrevious"
+      >
+        <b-icon-chevron-left />
+      </b-button>
+      <b-button
+        :disabled="!hasNext"
+        variant="outline-light"
+        @click="goNext"
+      >
+        <b-icon-chevron-right />
+      </b-button>
+    </b-button-group>
+
+    <b-button-toolbar ref="toolbar" class="my-4">
+      <b-button-group
+        v-for="(node, index) in getVisibleCards()"
+        :key="node.id"
+        :style="getButtonStyle(index)"
+      >
+        <b-button
+          :disabled="isButtonDisabled(index)"
+          variant="outline-light"
+          class="demo-button mx-1 mx-lg-3 mx-md-2 shadow-lg"
+          :style="{
+            width: `${buttonWidth}px`,
+            height: `${buttonHeight}px`,
+            backgroundImageFoo: `url(${placeholder})`
+          }"
+        >
+          {{ node.title }}
+        </b-button>
+      </b-button-group>
+    </b-button-toolbar>
+  </b-container>
+
+</template>
+
+
+<script>
+
+  import debounce from 'lodash/debounce';
+  import placeholder from '../assets/placeholder.png';
+
+  export default {
+    name: 'TagRow',
+    props: {
+      label: String,
+    },
+    data() {
+      return {
+        placeholder,
+        breakpoints: {},
+        margin: 0,
+        containerWidth: 0,
+        buttonWidth: 0,
+        buttonHeight: 0,
+        cardsPerRow: 5,
+        offset: 0,
+        nodes: [
+          { title: '1', id: '1' },
+          { title: '2', id: '2' },
+          { title: '3', id: '3' },
+          { title: '4', id: '4' },
+          { title: '5', id: '5' },
+          { title: '6', id: '6' },
+          { title: '7', id: '7' },
+          { title: '8', id: '8' },
+          { title: '9', id: '9' },
+          { title: '10', id: '10' },
+        ],
+      };
+    },
+    computed: {
+      hasPrevious: function() {
+        return this.offset > 0;
+      },
+      hasNext: function() {
+        return this.offset + this.cardsPerRow < this.nodes.length;
+      },
+    },
+    watch: {
+      containerWidth: function() {
+        this.debouncedUpdate();
+      },
+    },
+    created() {
+      window.addEventListener('resize', this.onResize);
+      const style = getComputedStyle(document.body);
+      this.breakpoints['xs'] = parseInt(
```

Comment:
Here we need the `radix` parameter in all the `parseInt`: https://eslint.org/docs/rules/radix

Reply:
Added!